### ISSUE TITLE
Can no longer insert additional columns in "custom" Row

### ISSUE THREAD

**Issue by rospencers** [liip/bootstrap-blocks-wordpress-plugin#111]
Using: 
- Bootstrap Blocks **4.2.1**, wiith Bootstrap Version **5.x** selected in Settings
- WordPress **6.0.1**

I am no longer able to insert additional columns into a **Row (Bootstrap)**, even when I've selected the "Custom" option in the _Change layout._ panel for the Row. 

I am able to delete Columns, unless there is only one left (which makes perfect sense). But I cannot add any additional Columns under any circumstances. 

I can find no "Add Column" option in the expansion menu for an existing Column in the List View, nor in the Column's options panel in the right sidebar. Likewise, the same are true for the Row block. The best I can do is select the Row's standardized "3 Columns (1:1:1)"...and then manually unlock them. But if I need 4 or more columns, I appear to be completely out of luck.

Adding _N_ Columns when in Row's Custom mode did work previously. I'm hoping I'm just missing something I should be noticing.

#### 1. Definitely the _Bootstrap_ Row block. :-) With "Custom" option selected.

![image](https://user-images.githubusercontent.com/24686373/181413035-e3cba546-c0fc-49e9-acbd-9a48cfcdb6fb.png)

#### 2. Column's menu no longer has an "Add Column" option (but does have "Remove"):

![image](https://user-images.githubusercontent.com/24686373/181411610-32e92c3e-d899-4270-b97f-b3a9d6f09c21.png)

#### 3. Not in Column's options panel either:

![image](https://user-images.githubusercontent.com/24686373/181411758-4ffb3ec6-3f43-4a3b-81e6-919d5856daff.png)

#### 4. Not in Row's menu.:

![image](https://user-images.githubusercontent.com/24686373/181411812-f18e763d-8f8b-4676-8f2c-ab5ecc877063.png)

#### 5. And not in Row's options panel (everything expanded):

![image](https://user-images.githubusercontent.com/24686373/181411895-dd1099b3-3952-4d16-b21c-ac2169501f1f.png)

**Comment by tschortsch** [comment]
Hi @rospencers. I sadly couldn't reproduce your issue in a fresh WordPress 6.0.1 setup. I inserted the following block combination: Container (Bootstrap) > Row (Bootstrap) > Columns (Bootstrap). I've set the row to use the custom template. As soon as it's selected I see the little plus sign on the lower left of the row block. Screenshot:
<img width="1428" alt="image" src="https://user-images.githubusercontent.com/1061850/181837813-5264bd72-a1c3-4a10-8eef-62413a22de05.png">

Maybe it has to something to do with the inner blocks you inserted inside the Column blocks. Could it be the case that their content overlaps the plus sign? Or maybe it's an issue with another editor plugin which interferes with the styling of the plus sign?
